src/vm.c

```diff
@@ -103,14 +93,14 @@ int v7_is_error(struct v7 *v7, val_t v) {
 }
 
 V7_PRIVATE val_t v7_pointer_to_value(void *p) {
-  return ((uint64_t) (uintptr_t) p) & ~V7_TAG_MASK;
+  return ((uint64_t)(uintptr_t) p) & ~V7_TAG_MASK;
 }
 
 V7_PRIVATE void *v7_to_pointer(val_t v) {
   struct {
-    uint64_t s:48;
+    uint64_t s : 48;
   } h;
-  return (void *) (uintptr_t) (h.s = v);
+  return (void *)(uintptr_t)(h.s = v);
```

Comment:
yep, I guess `clang-format -style=Google` should be the way to go.

One thing I like about Go is that you don't have to worry about formatting, like it or not.

clang-format does have a `SpaceAfterCStyleCast` option, but it's not there in the version of clang-format-3.5, it's a relatively new addition:

(http://llvm.org/bugs/show_bug.cgi?id=19982, http://lists.cs.uiuc.edu/pipermail/cfe-commits/Week-of-Mon-20140901/114058.html)

We could enforce proper clang format formatting as build test and local hook